content/swift/concepts/structures/structures.md

```diff
@@ -14,7 +14,7 @@ CatalogContent:
   - 'paths/build-ios-apps-with-swiftui'
 ---
 
-Structures, or `structs`, are used to programmatically represent a real-life object in code. Structures are created with the `struct` keyword followed by its name and then body containing its properties and methods.
+**Structures**, or **`structs`**, are used to programmatically represent a real-life object in code. Structures are created with the `struct` keyword followed by its name and then the body, which contains its properties and methods.
```

Comment:
```suggestion
**Structures, or structs,** are used to programmatically represent a real-life object in code. Structures are created with the `struct` keyword followed by its name and then the body, which contains its properties and methods.
```

I would remove the backticks to mitigate confusion over whether `structs` (plural) is a keyword vs. the single `struct`.